Modules/SubModule_Import/xASL_imp_NII2BIDS.m

```diff
@@ -56,9 +62,7 @@
 	datasetDescription = xASL_bids_CreateDatasetDescriptionTemplate(studyPar, x.Version);
```

Comment:
What if the `data_description` file already exists, will it keep it and not overwrite? Or merge?

Reply:
Not sure.

Reply:
Let's test this then. Same goes for `dataPar.json`, such files can already exist in the Root folder right? Especially in your use case, where you want to have processing options etc defined before running the conversion

Reply:
OK - Michael, can you please make this new issue? Also thinking of aggregating similar things to the same issue. And flavors related stuff has already an issue #886 that you can add stuff to.

Following the strategy of Henk - we only change import to module, all other fixes and features go to a new issue...

Reply:
good idea